cli/src/main/java/org/openjdk/skara/cli/GitJCheck.java

```diff
@@ -213,11 +239,98 @@ public static int run(Repository repo, String[] args) throws IOException {
             }
         }
 
+        var staged = arguments.contains("staged");
+        var working_tree = arguments.contains("working-tree");
```

Comment:
Please use camelCase for local variables.
```suggestion
        var workingTree = arguments.contains("working-tree");
```

Reply:
Will change it